selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java

```diff
@@ -88,19 +86,27 @@ public void createAndDeleteProjectTest() throws ExecutionException, InterruptedE
     projectSourcePage.selectSample(WEB_JAVA_SPRING);
     projectSourcePage.selectSample(CONSOLE_JAVA_SIMPLE);
     projectSourcePage.clickOnAddProjectButton();
+    projectSourcePage.clickOnAddOrImportProjectButton();
+    projectSourcePage.selectSample(WEB_JAVA_SPRING);
```

Comment:
it seems we are doing it twice.
Could you, please, leave a comment in the code about why we need to import the project and to select the _web java spring_ sample again?

Reply:
For the second creation 'web-java-spring-1' project will be created. I will add comment about it.